Typo in sp_arith.h.

Make build with NTL_LEGACY_SP_MULMOD=on.
diff -druN a/include/NTL/sp_arith.h b/include/NTL/sp_arith.h
--- a/include/NTL/sp_arith.h	2015-05-23 07:44:52.000000000 -0700
+++ b/include/NTL/sp_arith.h	2015-07-07 03:57:28.254261864 -0700
@@ -896,7 +896,7 @@
 
 static inline long MulMod2(long a, long b, long n, wide_double bninv)
 {
-   return MulMod2_legacy(a, b, n, ninv);
+   return MulMod2_legacy(a, b, n, bninv);
 }
 
 
